src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public FigureDrawable getRandomFigure() {
+        int value = random.nextInt(4);
```

Comment:
All magic numbers in your code should be constants. You should define a constant for the maximum number of figures. Refer to the checklist for the constant naming convention.